controllers/core/node_controller.go

```diff
@@ -113,7 +112,7 @@ func (r *NodeReconciler) SetupWithManager(mgr ctrl.Manager, healthzHandler *rcHe
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&corev1.Node{}).
 		WithOptions(controller.Options{MaxConcurrentReconciles: MaxNodeConcurrentReconciles}).
-		Owns(&v1alpha1.CNINode{}).Complete(r)
+		Complete(r)
```

Comment:
All this change does is doubles `MaxNodeConcurrentReconciles` since you're making two controllers. Have you thought about tuning the max concurrency?